arcgis10_mapping_tools/MapAction/MapAction/Utilities.cs

```diff
@@ -172,12 +204,11 @@ public static string createXML(Dictionary<string, string> usDict, string rootEle
         #endregion
 
         #region Public method getOperationConfigValues
-        //Returns a dictionary of the operation_config.xml elements and values
-        public static Dictionary<string, string> getOperationConfigValues(string path = null)
+        public static OperationConfig getOperationConfigValues(string path = null)

```

Comment:
see my separate conversation, as this reflects on the whole approach we take to refactoring, but i have some qualms about starting "new" (or at least, replacement), static methods, when what we are (i guess) hoping to end up with is some kind of "mission metadata" object on which this would perhaps be a method, rather than just being a static method in a utilities class. Personally, my approach would be to start by stubbing out those classes and then adding the methods to them there